manifest_staging/charts/secrets-store-csi-driver/values.yaml

```diff
@@ -22,6 +25,9 @@ linux:
       #   memory: "512Mi"
     annotations: {}
     podLabels: {}
+  # The secrets-store container security context
+  securityContext:
+    privileged: true
```

Comment:
`privileged: true` is **required** for the driver to function correctly. Moving this to `values.yaml` could cause the driver functionality to fail if not set correctly by the user.

`privileged: true` should instead remain in the manifest and this field should append to the existing `securityContext`.